### PR TITLE
SB-1090 Make AuthorizationConfigProvider thread-safe and access/store…

### PR DESCRIPTION
**Pull request by fuss86**
… authorization-configuration in the database

### CODE REVIEW

File: strongbox-web-core/src/test/resources/temp/metadata-faa-maven-plugin/META-INF/maven/plugin.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><plugin>
```
Comment (carlspring) [inline strongbox-web-core/src/test/resources/temp/metadata-faa-maven-plugin/META-INF/maven/plugin.xml]: What are these two XML-s for here? Are they related to the PR?

**Comment by carlspring** [comment]
Thanks for this PR! It's ready for merging, I just have a question about a few of the resources and then I believe we can merge this.

File: strongbox-web-core/src/test/resources/temp/metadata-faa-maven-plugin/META-INF/maven/plugin.xml (hunk 1 above)
Comment (fuss86) [reply strongbox-web-core/src/test/resources/temp/metadata-faa-maven-plugin/META-INF/maven/plugin.xml]: It's some test rubbish - haven't noticed it, sorry :) Removed.

**Comment by fuss86** [comment]
It's ready to be merged :)

**Comment by carlspring** [comment]
Thanks!

**Pull request merged**